set-definitions/pom.xml

```diff
@@ -29,5 +29,19 @@
 			<artifactId>commons-definitions</artifactId>
 			<version>${version.commonsApi}</version>
 		</dependency>
+
+        <!-- We need to add jackson databind for Deserializer, hence to be in sync
+           Override the jackson version present in api-commons-->
```

Comment:
jackson dependencies must not be added to the definitions module, but to the client.

Reply:
removed